### PR TITLE
Bug 1364204 disconnected image and tag fix

### PR DESCRIPTION
**Pull request by bfallonf**
As per: https://bugzilla.redhat.com/show_bug.cgi?id=1364204

FYI @rjhowe 

I'll ask for an ack in the BZ.

### CODE REVIEW

**Comment by rjhowe** [comment]
I like how this is vague in that we are not listing all possible tags.  I would though add some disclaimer regarding the use of quickstart templates, saying that many will not work out of the box with disconnected installs due to requiring certain tags and external repo accesses being needed. (maven,github, etc...)

**Comment by bfallonf** [comment]
@rjhowe Thanks for the look. I see what you mean. One of the intro paras stated something similar, but I've arranged it a little to point this out.

Can I get an a final ack this is up to what you're asking for? (Starting with "Also, when building applications...) Also, if there's any final comments? I can move ahead with this after that.

Big thanks.

**Comment by rjhowe** [comment]
@bfallonf LGTM

**Comment by bfallonf** [comment]
Thanks @rjhowe 

@tpoitras @ahardin-rh @adellape Available for peer review, please!

**Comment by bfallonf** [comment]
[rev_history]
|xref:../install_config/install/disconnected_install.adoc[Installing -> Disconnected Install]
|Fixed the tag references of images to be more generic.
%

File: install_config/install/disconnected_install.adoc [hunk 1]
```diff
@@ -38,13 +38,16 @@ internal Git repository is available that can host source code and this
...
+Also, when building applications in {product-title}, your build may have some
+external dependencies, such as a Maven Repository or Gem files for Ruby
+applications. For this reason, and because they might require certain tags, many
+of the Quickstart templates offered by {product-title} may not work on a
+disconnected environment. However, while Red Hat Docker images try to reach out
+to external repositories by default, you can configure {product-title} to use
+your own internal repositories. For the purposes of this document, we assume
+that such internal repositories already exist and are accessible from the
+{product-title} nodes hosts. Installing such repositories is outside the scope
+of this document.
```
Comment (tpoitras) [inline install_config/install/disconnected_install.adoc]: Should there be a link to the existing docs for creating internal repos?

File: install_config/install/disconnected_install.adoc [hunk 2]
```diff
@@ -188,28 +191,37 @@ additional centralized log aggregation and metrics aggregation components:
...
+* mysql
+* nodejs
+* perl 
+* php 
+* postgresql 
+* python 
+* redhat-sso70-openshift
+* ruby
++
+Make sure to specify the correct tag specifying the desired version number. For
```
Comment (tpoitras) [inline install_config/install/disconnected_install.adoc]: The double "specify" makes my head spin. Is there a better way to phrase that?

File: install_config/install/disconnected_install.adoc [hunk 3]
```diff
@@ -599,19 +597,9 @@ earlier:
...
-# docker push $REGISTRY:5000/openshift/python-27-rhel7
-# docker push $REGISTRY:5000/openshift/python-34-rhel7
-# docker push $REGISTRY:5000/openshift/ruby-22-rhel7
-# docker push $REGISTRY:5000/openshift/nodejs-010-rhel7
+# docker push $REGISTRY:5000/openshift/webserver30-tomcat7-openshift:1.1
+# docker push $REGISTRY:5000/openshift/webserver30-tomcat7-openshift:1.2
+# docker push $REGISTRY:5000/openshift/webserver30-tomcat7-openshift:latest
 ----
 
 . Verify the that all the image streams now have the tags populated:
```
Comment (tpoitras) [inline install_config/install/disconnected_install.adoc]: "the that"

**Comment by tpoitras** [comment]
@bfallonf made a few comments, nothing major. Other than those minor things, LGTM. :mortar_board:

File: install_config/install/disconnected_install.adoc (hunk 1 above)
Comment (bfallonf) [inline install_config/install/disconnected_install.adoc]: Hmm that may be an omission worthy of more research. I'll send some emails.

**Pull request merged**